src/content/1.7/modules/concepts/doctrine.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Doctrine
+weight: 5
+---
+
+# Doctrine
+{{< minver v="1.7.6" title="true" >}}
+
+From the 1.7.6 version of PrestaShop we integrate support of Doctrine services and entities for modules. Doctrine is
+a powerful ORM allowing you to manage your database data via objects. It offers an abstract layer allowing you to perform
+insert/update actions via a simple `$entity->setData('update')` call. But you can also create your own repositories to fetch
+specific data via left join, add conditions and so on...
+
+Doctrine is the default ORM integrated with Symfony which is why we added it for modules in legacy context. If you want
+more details about Doctrine and its features you can check [their documentation](https://www.doctrine-project.org/projects/doctrine-orm/en/2.5/index.html)
+or the [Symfony documentation](https://symfony.com/doc/3.4/doctrine.html). PrestaShop is currently using Doctrine version 2.5.
+
+## Module integration
+
+We meant to make the Doctrine integration the simplest possible, which is why we favored the [annotation mapping](https://symfony.com/doc/3.4/doctrine.html#add-mapping-information)
+which allows you to specify your database structure directly in your entity classes. We also used a simple convention, you simply
+need to put your entities in the `src/Entity` folder of your module and PrestaShop will automatically scan them for all **installed**
+modules.
+
+To work correctly you will need to define a namespace for your entity classes using Composer, you can find more details
+about namespace setup in [Setup composer][setup-composer] page.
+
+[setup-composer]: {{< ref "/1.7/modules/concepts/services/_index.md#setup-composer" >}}
+
+## Define an Entity
+
+Here is a simple example of a `ProductComment` entity:
+
+```php
+// modules/yourmodule/src/Entity/ProductComment.php
+namespace PrestaShop\Module\YourModule\Entity;
```

Comment:
```suggestion
namespace YourCompany\Module\YourModule\Entity;
```
We use `PrestaShop\Module` for native modules.
It's maybe better to show something different.

Reply:
Nice catch! Done!